### PR TITLE
Update strings-nl.xml

### PR DESCRIPTION
**Pull request by KevinDenys**
Updated Dutch Translations

### CODE REVIEW

**Comment by jamorham** [comment]
I'm still trying to get another dutch speaker to review these changes.

File: app/src/main/res/values-nl/strings-nl.xml [hunk 1]
```diff
@@ -7,21 +7,21 @@
 	<string name="menu_stop">Stop</string>
 	<string name="menu_refresh">Verversen</string>
 	<string name="menu_export_database">Export databank</string>
-	<string name="menu_export_csv_sidiary">Export CSV (dochteronderneming-formaat)</string>
-	<string name="menu_import_db">Import-database</string>
+	<string name="menu_export_csv_sidiary">Exporteer CSV (dochteronderneming-formaat)</string>
```
Comment (kretep) [inline app/src/main/res/values-nl/strings-nl.xml]: I would change this to "Exporteer CSV (SiDiary format)", as SiDiary doesn't really translate

File: app/src/main/res/values-nl/strings-nl.xml [hunk 2]
```diff
@@ -7,21 +7,21 @@
...
+	<string name="menu_import_db">Importeer-database</string>
```
Comment (kretep) [inline app/src/main/res/values-nl/strings-nl.xml]: I would change this to "Importeer database" (without dash)

File: app/src/main/res/values-nl/strings-nl.xml [hunk 3]
```diff
@@ -7,21 +7,21 @@
 	<string name="menu_stop">Stop</string>
 	<string name="menu_refresh">Verversen</string>
 	<string name="menu_export_database">Export databank</string>
```
Comment (kretep) [inline app/src/main/res/values-nl/strings-nl.xml]: To stay consistent, I would change this to "Exporteer database" (I guess databank is valid Dutch, but I have never heard anybody use it)

**Pull request merged**